lib/resource-mapper.js

```diff
@@ -8,7 +8,12 @@ const readdir = promisify(fs.readdir)
 // following the principles of the “sweet spot” discussed in
 // https://www.w3.org/DesignIssues/HTTPFilenameMapping.html
 class ResourceMapper {
-  constructor ({ rootUrl, rootPath, includeHost, defaultContentType = 'application/octet-stream' }) {
+  constructor ({
+    rootUrl = 'https://localhost:8443/',
+    rootPath = process.cwd(),
+    includeHost = false,
+    defaultContentType = 'application/octet-stream'
+  }) {
```

Comment:
Is it wise to have these defaults? (just asking)

Reply:
No strong opinion here, do you propose to remove/change them?